src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx

```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
+
+### You're running services in orchestrated environments [#orchestrated]
+
+If you are running services in containerized, orchestrated environments, choose your update scenario:
+
+* <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="Kubernetes" alt="Kubernetes" src={kubernetesLogo}/> [Update on-host integrations on Kubernetes](/docs/integrations/kubernetes-integration/link-apps-services/monitor-services-running-kubernetes)
+* <img style={{ width: '32px', height: '32px', verticalAlign: 'middle'}} class="inline" title="ECS" alt="ECS" src={ecs}/> [Update on-host integrations on Amazon ECS](/docs/integrations/host-integrations/host-integrations-list/monitor-services-running-amazon-ecs) (EC2 launch type)
+
+Our on-host integrations and the ECS and Kubernetes integration can run together; see [Intro to Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) and [Intro to ECS integration](/docs/integrations/elastic-container-service-integration/get-started/introduction-amazon-ecs-integration) to learn more.
+
+### You're running on-premises services or services on single VMs [#standard]
+
+If you are running services without orchestration or on-premises, choose your scenario:
+
+<CollapserGroup>
+  <Collapser
+    id="windows"
+    title={<>Update using MSI files (<img src={windows} title="Windows Server" alt="Windows Server" style={{ height: '32px', width: '32px', verticalAlign: 'middle' }}/> Windows Server)</>}
+  >
+    Here are two approaches, depending on the integration you're using.
+
+    **On-host Microsoft Windows services integration**
+
+    The [Microsoft Windows services integration](/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration/) is bundled with the New Relic infrastructure agent, so to update the integration, you need to update the infrastructure agent. See [Update the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/update-infrastructure-agent).
+
+    **All other on-host Microsoft Windows integrations**
+
+    You can update integrations for services running on Windows Server with our MSI installer or the official zip files. We recommend the MSI installation.
+
+    1. Download the latest MSI installer image [from our repository](http://download.newrelic.com/infrastructure_agent/windows/integrations/).
+    2. In an admin account, run the install script using an absolute path.
+
+      ```
+      msiexec.exe /qn /i <var>PATH\TO\INSERT_THE_INTEGRATION_NAME.msi</var>
+      ```
+    3. Change `C:\Program Files\New Relic\newrelic-infra\integrations.d\integration-name-config.yml.sample` to `integration-name-config.yml`, and edit according to your needs.
```

Comment:
This step 3 can be removed. This is only needed for a first install. When updating the installer will keep the existing `integration-name-config.yml` file